kobiton-rest-api/get-devices-info/shell/check-device-status.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+function jsonValue() {
```

Comment:
Use `echo '{"test":1,"test2":2}' | python -mjson.tool` since we're using bash where python is available by default